### DISCUSSION TITLE
Do thumbnails not work on GIF's?

### DISCUSSION THREAD

**Discussion by Warpzal** [pocketbase/pocketbase#507] [Q&A]
I've noticed that thumbnails don't load for gifs like they do for other image formats, wanted to know if this was intended.

**Comment by ganigeorgiev** [comment]
Yes, currently only images with the following mimetypes are allowed to have thumbs - `image/png`, `image/jpg`, `image/jpeg`.

I'll consider in a future release to add `image/gif` to the supported thumbs list.
